pkg/crc/machine/bundle/repository.go

```diff
@@ -117,6 +120,23 @@ func (bundle *CrcBundleInfo) createSymlinkOrCopyOpenShiftClient(ocBinDir string)
 	return os.Symlink(ocInBundle, ocInBinDir)
 }
 
+func (bundle *CrcBundleInfo) createSymlinkOrCopyPodmanClient(ocBinDir string) error {
+	podmanInBundle := bundle.GetPodmanPath()
+	if podmanInBundle == "" {
+		return nil
+	}
+	podmanInBinDir := filepath.Join(ocBinDir, filepath.Base(podmanInBundle))
```

Comment:
why we want to have podman in `ocBinDir` just because symlink pupose?

Reply:
Because if user has ocBinDir in their PATH, podman will already be available for them without extra action.

We could rename oc binary directory but it would break some users, etc.